token-lending/cli/src/main.rs

```diff
@@ -592,6 +603,8 @@ fn main() {
             let borrow_fee_wad = (borrow_fee * WAD as f64) as u64;
             let flash_loan_fee_wad = (flash_loan_fee * WAD as f64) as u64;
 
+            let liquidity_fee_receiver_keypair = Keypair::new();
```

Comment:
this seems wrong shouldn't it be the thing you passed in?

Reply:
For adding a new reserve, we just make a new one, because we also just make the fee receiver account. in this command.
The thing passed in is for updating the reserve